File: Conditional_Statements_Advanced_Exercise/journey.py
budget = float(input())
season = input()

price = 0
destination = ""
trip = ""

if budget <= 100:
    if season == "summer":
        price = budget * 0.30
    elif season == "winter":
        price = budget * 0.70
elif budget <= 1000:
    if season == "summer":
        price = budget * 0.40
    elif season == "winter":
        price = budget * 0.80
else:
    if season == "summer" or season == "winter":
        price = budget * 0.90

if budget <= 100:
    destination = "Bulgaria"
    print(f"Somewhere in {destination} ")
elif budget <= 1000:
    destination = "Balkans"
    print(f"Somewhere in {destination} ")
else:
    destination = "Europe"
    print(f"Somewhere in {destination} ")

if destination == "Europe":
    trip = "Hotel"
elif season == "summer":
    trip = "Camp"
else:
    trip = "Hotel"
print(f"{trip} - {price:.2f}")

File: C++/Basics/ConditionalStatements_Advanced_Exercise/Journey/Journey.cpp
#include <iostream>
using namespace std;

int main()
{
	double budget, price;
	string season;

	string destination;
	string typeOfPlace;

	cin >> budget >> season;

	if (budget <= 100) {
		destination = "Bulgaria";

		if (season == "summer") {
			typeOfPlace = "Camp";
			price = budget * 0.30;
		}
		else if (season == "winter") {
			typeOfPlace = "Hotel";
			price = budget * 0.70;
		}
	}
	else if (budget <= 1000) {
		destination = "Balkans";

		if (season == "summer") {
			typeOfPlace = "Camp";
			price = budget * 0.40;
		}
		else if (season == "winter") {
			typeOfPlace = "Hotel";
			price = budget * 0.80;
		}
	}
	else if (budget > 1000) {
		destination = "Europe";
		typeOfPlace = "Hotel";
		price = budget * 0.90;
	}

	cout.setf(ios::fixed);
	cout.precision(2);

	cout << "Somewhere in " << destination << endl;
	cout << typeOfPlace << " - " << price << endl;
}